app/src/components/Modals/WalletModal/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import {
+  MetamaskModalSvg,
+  WalletConnect,
+  CoinbaseSvg,
+  ArgentSvg,
+  TrustSvg,
+  LedgerLiveSvg,
+  ImWalletSvg,
+  SteakWalletSvg
+} from 'components/IconSvg/WalletSvg'
+import CloseIcon from '../../IconSvg/CloseIcon'
+import injected from 'config/Walletconnectors'
+import { useWeb3 } from 'hooks'
+import { IModalProps } from './interface'
+import {
+  ModalWrapper,
+  ModalContainer,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalItem,
+  ModalBackDrop,
+  UnavailableWalletHeader
+} from './styled'
+
+const availableWallets = [
+  {
+    icon: <MetamaskModalSvg />,
+    label: 'MetaMask'
+  }
+]
+
+const comingSoonWallets = [
```

Comment:
We don't need separate structure for this, you can just add a boolean to the previous data to determine if it's not available yet